wp-content/plugins/Contact Us Plugin/contact-us.php

```diff
@@ -0,0 +1,60 @@
+<?php
+	/*
+	Plugin Name: Contact Us plugin
+	Description: Building a Contact us plugin for demosite.
```

Comment:
But sir for plugin name we always use in header on commenting name on file. otherwise plugins can't be active in wordpress.